docs/visualization/visualize_2p_responses.ipynb

```diff
@@ -38,7 +38,7 @@
   },
```

Comment:
<p>Might be better to describe this in the markdown above. Like "the cell below retrieves blah blah from blah as well as the blah blah from the blah". I think typically, its better to aim to conceptually describe blocks of code unless the purpose/relevance of one line is particularly confusing.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>